src/mkpsxiso/main.cpp

```diff
@@ -1127,6 +1264,49 @@ static bool ParseFileEntry(iso::DirTreeClass* dirTree, const tinyxml2::XMLElemen
 				printf( "ERROR: DA audio file(s) specified but no CUE sheet specified.\n" );
 				return false;
 			}
+			trackid = dirElement->Attribute(xml::attrib::TRACK_ID);
+			if ( trackid == nullptr )
+			{
+				printf( "ERROR: DA audio file(s) does not have an associated CDDA track [trackid]\n" );
+				return false;
+			}
+			// locate the node containing the tracks
+			const tinyxml2::XMLElement *isoElement;
+			for( const tinyxml2::XMLElement *parent = (tinyxml2::XMLElement *)dirElement->Parent(); ; parent = (tinyxml2::XMLElement *)parent->Parent())
+			{
+				if(parent == nullptr)
+				{
+					printf( "ERROR: locating DA file\n" );
+					return false;
+				}
+				if(strcmp(parent->Name(), xml::elem::ISO_PROJECT) == 0)

```

Comment:
Should probably be case-insensitive again.

Reply:
Fixed in G4Vi@48bec80